worker-sys/src/types/version.rs

```diff
@@ -0,0 +1,15 @@
+use wasm_bindgen::prelude::*;
+
+/// This type was created to support https://developers.cloudflare.com/workers/runtime-apis/bindings/version-metadata/
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(extends=js_sys::Object)]
+    #[derive(Clone, PartialEq, Eq)]
+    pub type CfVersionMetadata;
```

Comment:
This type also has a [`timestamp`](https://workers-types.pages.dev/#WorkerVersionMetadata.timestamp) field defined for when the Worker was uploaded.

Reply:
Thanks, had no idea this page existed! I've added the timestamp